cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -691,6 +692,23 @@ def create_security_group_rule(self, group_id, tenant_id, protocol='tcp',
                                             "port_range_max": port_range_max}}
         self.neutronclient.create_security_group_rule(sec_rule)
 
+    def create_cinder_volume_types_on_all_clouds(self):
+        self.create_cinder_volume_types(self.config.cinder_volume_types)
+        self.dst_cloud.create_cinder_volume_types(
+            self.config.cinder_volume_types)
+
+    def create_cinder_volume_types(self, volume_types_list):
+        self.log.debug("Creating cinder volume types on '%s' cloud",
+                       self.cloud_prefix)
+        for volume_type in volume_types_list:
+            try:
+                vt = self.cinderclient.volume_types.create(volume_type['name'])
+                vt.set_keys(volume_type['metadata'])
+            except cd_exceptions.ClientException as e:
+                self.log.debug(e)
+                self.log.warning("Volume type '%s' already exists",
```

Comment:
`ClientException` probably represents more than one exception type, let's modify warning message to `"Failed to create volume type '%s', error: %s", error`